src/helpers/helpers.math.js

```diff
@@ -103,7 +103,7 @@ export function toDegrees(radians) {
  */
 export function _decimalPlaces(x) {
 	if (!isFiniteNumber(x)) {
-		return;
+		return NaN;
```

Comment:
`undefined` makes more sense for `_decimalPlaces('not a number')`? not having any strong opinions about this though.

Reply:
Yeah, I kind of agree. The issue is then that we would have to change the return type of the method to `{number=}`, which would cause us to have to explicitly handle `undefined` results, which got really gross. This was much cleaner overall